spec/features-json/setting_json_controller_spec.rb

```diff
@@ -66,5 +67,14 @@
 
       expect(FastlaneCI::Services.setting_service.find_setting(setting_key: metrics_key.to_sym).value).to eq(nil)
     end
+
+    it "returns an error if key can't be found" do
```

Comment:
I think we also test http status code (`400` in this case?) because client-side might have some status code logic.

Reply:
Yeah good point, will update 👍